### PR TITLE
Updates to specifications and vocabulary content

### PR DESCRIPTION
**Pull request by jamsden**

### CODE REVIEW

**Comment by jamsden** [comment]
Updates for configuration management and tractor resource at oasis standards will be done as soon as they are published

File: content/specifications/_index.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Note: The open-services.net specifications that are being migrated to OASIS by t
 | [Quality Management 2.1 OS](https://docs.oasis-open-projects.org/oslc-op/qm/v2.1/os/quality-management-spec.html) | **OSLC Open Project** | Defines the OSLC services and vocabulary for the Quality Management domain. |
 | [Requirements Management 2.1 OS](https://docs.oasis-open-projects.org/oslc-op/rm/v2.1/os/requirements-management-spec.html) | **OSLC Open Project** | Defines the OSLC services and vocabulary for the Requirements Management domain. **OASIS Standard** |
 | [Change Management 3.0 OS](https://docs.oasis-open-projects.org/oslc-op/cm/v3.0/os/change-mgt-spec.html) | **OSLC Open Project** | Defines the OSLC services and vocabulary for the Change Management domain. **OASIS Standard** |
-| [Tracked Resource Set 3.0 PS01](https://docs.oasis-open-projects.org/oslc-op/trs/v3.0/ps01/tracked-resource-set.html) |  **OSLC Open Project**  | Allows servers to expose a set of resources whose state can be tracked by clients. |
```
Comment (berezovskyi) [inline content/specifications/_index.md:34]: Please do not edit entries. All previously published versions should remain accessible. Instead, add new rows and move the old ones to the "Obsolete" table down below.

**Review by berezovskyi — CHANGES_REQUESTED** [review]
Looks good except for the requested change on the table to recover old entries.

File: content/specifications/_index.md (hunk 1 above)
Comment (jamsden) [reply content/specifications/_index.md:34]: The obsolete table is full of a lot of not very useful information that could be confusing to people looking for OSLC OP specifications. And in addition, the history of of published specifications is already managed by the specifications themselves with their previous version fields which have links to the files on oasis. So this obsolete table seems redundant for many of those revised documents. What if we only include documents that are not part of published by oasis in this table?

Comment (berezovskyi) [reply content/specifications/_index.md:34]: I don't really share this view as I think that "Obsolete publications" heading removes any confusion. However, it's up to you and Jad to decide whether to remove the "Obsolete publications" table altogether. Until then, I suggest to keep tables updated as originally intended.

**Review by berezovskyi — APPROVED** [review]

File: content/specifications/_index.md (hunk 1 above)
Comment (jamsden) [reply content/specifications/_index.md:34]: fixed

**Pull request merged**